MantidPlot/src/ProjectRecoveryThread.cpp

```diff
@@ -0,0 +1,326 @@
+#include "ProjectRecoveryThread.h"
+
+#include "ApplicationWindow.h"
+#include "Folder.h"
+#include "ProjectSerialiser.h"
+#include "globals.h"
```

Comment:
Do we need anything our of the globals header?

Reply:
Not in this PR, but in the next one (for project recovery) the load method in `ProjectSerialiser` requires the mantid version as a parameter

Reply:
I see. In that case we should use the `MantidKernel/MantidVersion.h` header which is the proper mantid versioning information. That file is from the original `qtiplot` and so it won't quite give the numbers you would expect.

Reply:
Okay I'll pick that up in the loading PR, as this is only relevant there.
I've removed the header in the mean time

Reply:
:+1: